plugins/MauticCrmBundle/Api/SalesforceApi.php

```diff
@@ -15,6 +15,9 @@ class SalesforceApi extends CrmApi
     protected $requestSettings = [
         'encode_parameters' => 'json',
     ];
+
+    const CURL_SSLVERSION_TLSv1_1 = 1;
```

Comment:
I thought so too, but I am not sure why it's being used like this. I thought maybe setting the default value (0)